### PR TITLE
Add PHP autofix to pre-commit hook

### LINKED ISSUE #2021: PHP Linter autofix should be run as a pre-commit hook
1. Checkout master: `git checkout master`.
2. Run PHP Lint autofix: `npm run lint:php-fix`.
3. See what changed: `git status`.
4. Notice we have several files with issues.
5. We should run PHP Lint autofix as a pre-commit hook to prevent this happening in the future.

### PR DESCRIPTION
**Pull request by Aljullu**
Fixes #2021.

Add PHP Lint autofix to pre-commit hook.

### Detailed test instructions:
- Edit any PHP file and add something like this:
```PHP
if ( 1 == 1 ) {
   return; // Spaces instead of tabs
}
```
- With git, add that file and do a commit.
- Notice the spaces instead of tabs issue if automatically fixed.
- Notice not adding a period in the comment prevents you from committing and you manually have to fix that issue.

### CODE REVIEW

**Review by psealock — COMMENTED** [review]
I get a different message when committing your code snippet

Screen Shot 2019-04-10 at 9 53 08 AM

When I remove the comment, the commit is allowed to happen. `npm run lint:php` does catch the errors.

Screen Shot 2019-04-10 at 9 54 59 AM

Is this not what you're experiencing?

**Comment by Aljullu** [comment]
Thanks for the review @psealock! You're right, my example was wrong. Strict comparisons is a warning instead of an error, so it doesn't block commiting. I think that's to be expected and we are currently not catching warnings either.

I updated the steps to reproduce and added some code to check if PHP_CodeSniffer is installed. Feel free to take another look. :+1:

File: bin/pre-commit-hook.js [hunk 1]
```diff
@@ -91,30 +91,55 @@ if ( lintResult.status ) {
...
 	try {
-		execSync( `php -l -d display_errors=0 ${ file }` );
-	} catch( err ) {
-		hasPhpLintErrors = true;
+		// Check if PHP_CodeSniffer is installed.
+		execSync( `./vendor/bin/phpcbf -h` );
+		execSync( `./vendor/bin/phpcs -h` );
+	} catch( e ) {
+		console.log(
+			'PHP_CodeSniffer is not installed. ' +
```
Comment (joshuatf) [inline bin/pre-commit-hook.js]: ❤️

**Review by joshuatf — APPROVED** [review]
Sweet.  Testing well for fixable phpcbf items.  LGTM if @psealock doesn't have any other concerns.

**Review by psealock — APPROVED** [review]
Oh right, I didn't see those were warnings. LGTM

**Pull request merged**